frontend/src/components/Register.scss

```diff
@@ -0,0 +1,43 @@
+input
+{
+    margin-bottom: 5px;
+}
+
+.register
+{
+    font-size: 17px;
+    display:flex;
+    justify-content: center;
+}
+
+.user-names-left
+{
+    margin-right:30px;
+}
+
+.user-names
+{
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content:space-between;
+    float: left;
+    margin-bottom: 10px;
+}
+
+.button
+{
+    color: rgb(25, 136, 240);
+    background-color: rgb(40, 40, 40);
+    outline: none;
+    border: none;
+    padding: 5px 20px;
+    text-transform: uppercase;
+    border-radius: 2px;
+    cursor: pointer;
+}
+
+.register-button
+{
+    display:flex;
+    justify-content: flex-end;
+}
```

Comment:
I will have to import the prelude.scss into this file

Reply:
Sounds good, your declaration should be a simple `@import /* insert relative path to access the prelude */` (probably something like `../styles/prelude.scss`)!